joint_trajectory_controller/include/joint_trajectory_controller/joint_trajectory_controller.hpp

```diff
@@ -145,6 +146,11 @@ class JointTrajectoryController : public controller_interface::ControllerInterfa
   // There should be PID-approach used as in ROS1:
   // https://github.com/ros-controls/ros_controllers/blob/noetic-devel/joint_trajectory_controller/include/joint_trajectory_controller/hardware_interface_adapter.h#L283
   bool use_closed_loop_pid_adapter = false;
+  using PidPtr = std::shared_ptr<control_toolbox::Pid>;
+  std::vector<PidPtr> pids_;
+  std::chrono::steady_clock::time_point last_update_time_;
+  std::vector<double> velocity_ff_;
+  std::vector<double> command_;
```

Comment:
Add doc string comment and rename to avoid confusion in the future.

```suggestion
  /// reserved storage for result of the command when closed loop pid adapter is used
  std::vector<double> tmp_command_;
```

Reply:
ros-controls/ros2_controllers@09e6954